mage/changelog.md

```diff
@@ -4,6 +4,21 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v1.5 - Dec 20, 2022
+
+### Major Features and Improvements
+
+- Added support for ancestors, descendants, topological sort, connecting nodes, and chaining nodes.
```

Comment:
now you can do this this this and this how?

Reply:
Fixed